build.gradle

```diff
@@ -25,15 +25,22 @@ test {
     useJUnitPlatform()
 }
 
-task compressAssignment(type: Zip) {
+task compressAssignment1(type: Zip) {
     archiveFileName = "Assignment1.zip"
     destinationDirectory = file("src/site/data/")
     from "src/site/data/Assignment1"
 }
 
+task compressAssignment2(type: Zip) {
+    archiveFileName = "Assignment3.zip"
+    destinationDirectory = file("src/site/data/")
+    from "src/site/data/Assignment3"
```

Comment:
Presumably this is supposed to be assignment 2

Reply:
what. the. f?